src/app/modules/main/profile_section/profile/models/profile_showcase_preferences_item.nim

```diff
@@ -0,0 +1,23 @@
+import app_service/service/profile/dto/profile_showcase_preferences
+
+type
+  ProfileShowcasePreferencesItem* = object of RootObj
+    showcaseKey*: string
+    showcaseVisibility*: ProfileShowcaseVisibility
+    showcasePosition*: int
```

Comment:
Do we then need the position? I think we talked about using the natural order at least in the UI, with the natural order it should be enough

Reply:
If we don't have plans to support diff changes, i can remove it. But just in case we will need it in the future it's worth having them in the model.

Reply:
@alexjba @micieslak wdyt?

Reply:
I think it's ok to have it in the preferences model. We already have code in the new models that depends on `showcasePosition`. IMO the contacts profile doesn't need the position and can use the natural order.